Add explaination on how to enable SOAR logs. (#1294)
MicroEJ/docs: ApplicationDeveloperGuide/soar.rst

   The Binary Code Verifier is enabled by default when building a Sandboxed Application, and disabled by default when building a Standalone Application.
   See :ref:`option_enable_bytecode_verifier` for more details.

> Enable SOAR Logs

Reviewer: ```suggestion
Enable SOAR Verbose Logs
```

----------------

> SOAR logs can be enabled by setting the ``execution.verbose`` system property to ``true``.

Reviewer: ```suggestion
SOAR logs can be enabled by setting the ``execution.verbose`` Application Option to ``true``.
```

As for :ref:`Application Options <application_options>`, with SDK 6 the system property must be prefixed with ``microej.option``.

For instance, to enable SOAR logs from command line::

    -Dmicroej.option.execution.verbose=true

or from ``gradle.properties`` file::

>    systemProp.microej.execution.verbose=true

Reviewer: ```suggestion
   systemProp.microej.option.execution.verbose=true
```
